### PR TITLE
FACES-2363 When value is empty, bridge throws java.net.MalformedURLException: invalid name/value pair: name=

### PR DESCRIPTION
**Pull request by stiemannkj1**

### CODE REVIEW

File: bridge-impl/src/main/java/com/liferay/faces/bridge/context/url/internal/BridgeURLBase.java [hunk 1]
```diff
@@ -444,9 +444,16 @@ protected boolean matchPathAndExtension(String file1, String file2) {
...
+						if (nameValueArray.length == 1) {
+
+							String name = nameValueArray[0];
+							requestParameters.add(new RequestParameter(name, ""));
+						}
+						else if (nameValueArray.length == 2) {
+
 							String name = nameValueArray[0];
 							String value = nameValueArray[1];
 							requestParameters.add(new RequestParameter(name, value));
```
Comment (stiemannkj1) [inline bridge-impl/src/main/java/com/liferay/faces/bridge/context/url/internal/BridgeURLBase.java]: @ngriffin7a,
**Side note:** Should we thrown a `MalformedURLException` when `"".equals(name)`?

**Comment by stiemannkj1** [comment]
Closing because we may need to do the same work in other places as well: https://github.com/liferay/liferay-faces/blob/master/bridge-impl/src/main/java/com/liferay/faces/bridge/context/map/internal/FacesViewParameterMap.java#L61.

Also, should we check for an empty `name`? For example, should we throw a `MalformedURLException` for the following URL?

    https://someurl.com/somepage.faces?name1=value1&=value2

**Comment by ngriffin7a** [comment]
@stiemannkj1 please fix in all places, including the one @vsingleton mentioned. If the parameter value is null/empty then that should *not* throw a MalformedURLException. However, if the parameter name is missing, meaning "=value2" then it should indeed throw a MalformedURLException.

**Comment by stiemannkj1** [comment]
@ngriffin7a, done: #803

**Pull request closed without merging**